src/client/filters/Mask.ts

```
@@ -0,0 +1,34 @@
+import Filter from './Filter';
+
+class Mask extends Filter {
+	constructor() {
+		super();
+	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)